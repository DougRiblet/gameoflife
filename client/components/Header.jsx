import React from 'react'

export default class Header extends React.Component {
  constructor (props) {
    super(props)
    this.state = {}
  }

  render () {
    return (
      <div>
        <h2>Conway's Game of Life</h2>
      </div>
    )
  }

}
