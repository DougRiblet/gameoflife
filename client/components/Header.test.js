import React from 'react'
import ReactDOM from 'react-dom'
import Header from './Header'
import renderer from 'react-test-renderer'

test('Header component displays header', () => {
  const component = renderer.create(
    <Header />
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})

describe('Header', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div')
        ReactDOM.render(<Header/>, div)
    })
})
