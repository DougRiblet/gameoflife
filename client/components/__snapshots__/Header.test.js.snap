// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component displays header 1`] = `
<div>
  <h2>
    Conway's Game of Life
  </h2>
</div>
`;
